README.md

```diff
@@ -34,7 +34,8 @@ See more in the [documentation](https://juliafolds.github.io/Transducers.jl/dev)
 ## Installation
 
 ```
-]add Transducers
+using Pkg
+Pkg.add("Transducers")
```

Comment:
Thank you for this. The cryptic Pkg markup is a tripping point for newbies, especially since pasting these commands into the REPL currently does not work (https://github.com/JuliaLang/julia/issues/35605).